sdk/communication/Azure.Communication.Sms/tests/SmsClientLiveTests.cs

```diff
@@ -30,15 +30,14 @@ public async Task SendingAnSmsMessage()
             try
             {
                 #region Snippet:Azure_Communication_Sms_Tests_SendAsync
-                SendSmsResponse result = await client.SendAsync(
-                   //@@ from: new PhoneNumber("+18001230000"), // Phone number acquired on your Azure Communication resource
-                   //@@ to: new PhoneNumber("+18005670000"),
-                   /*@@*/ from: new PhoneNumberIdentifier(TestEnvironment.PhoneNumber),
-                   /*@@*/ to: new PhoneNumberIdentifier(TestEnvironment.PhoneNumber),
+                SmsSendResult result = await client.SendAsync(
+                   from: TestEnvironment.PhoneNumber,
+                   to: TestEnvironment.PhoneNumber,
                    message: "Hi");
                 Console.WriteLine($"Sms id: {result.MessageId}");
                 #endregion Snippet:Azure_Communication_Sms_Tests_SendAsync
-                /*@@*/ Assert.IsFalse(string.IsNullOrWhiteSpace(result.MessageId));
+                /*@@*/
+                Assert.IsFalse(string.IsNullOrWhiteSpace(result.MessageId));
```

Comment:
seems a mistake. If this is supposed to be a region, `/*@@*/` should be on the same line